v2/definitions/VideoJobRequest.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  type: 'object',
+  required: ['video', 'fps', 'classes'],
+  properties: {
+    video: {
+      type: 'string',
+      description: 'URL of the video',
```

Comment:
Should we mention anything about accepted video formats?

Reply:
To be updated later when we know the video formats it can accept.